microsoft/testsuites/ltp/ltp.py

```diff
@@ -340,14 +345,31 @@ def _install(self) -> bool:
         self.node.tools[Mkdir].create_directory(top_src_dir, sudo=True)
         self.node.tools[Chmod].update_folder(top_src_dir, "a+rwX", sudo=True)
 
-        # clone ltp
-        git = self.node.tools[Git]
-        ltp_path = git.clone(
-            self.LTP_GIT_URL, cwd=PurePosixPath(top_src_dir), dir_name=top_src_dir
-        )
+        if self._tar_file_path:
+            self._log.debug(f"Use downloaded ltp tar: {self._tar_file_path}!")
+            self._remote_tar_path = self.get_tool_path(
+                use_global=True
+            ) / os.path.basename(self._tar_file_path)
+
+            mkdir = self.node.tools[Mkdir]
+            mkdir.create_directory(self._remote_tar_path.parent.as_posix())
```

Comment:
The remotecopy should create the path inside, so it doesn't need to create before copy. If it's not, please support it in remotecopy.